TransferMethod/Sources/Extensions/HyperwalletTransferMethodUI.swift

```diff
@@ -43,9 +50,9 @@ public extension HyperwalletUI {
         -> SelectTransferMethodTypeCoordinator {
```

Comment:
Update this method as below.
<img width="790" alt="Screenshot 2020-03-23 at 1 55 12 PM" src="https://user-images.githubusercontent.com/55200187/77296624-04757e00-6d0e-11ea-81b7-95b7218ad773.png">

Reply:
Updated